backend/index.js

```diff
@@ -3,6 +3,9 @@ const http = require("http");
 const app = express();
 const server = http.createServer(app);
 const cors = require("cors");
```

Comment:
All services must be imported by a controller. Separate the routes and controllers accordingly.